databricks/koalas/missing/frame.py

```diff
@@ -158,3 +157,12 @@ class _MissingPandasLikeDataFrame(object):
     select = unsupported_function('select', deprecated=True)
     set_value = unsupported_function('set_value', deprecated=True)
     to_panel = unsupported_function('to_panel', deprecated=True)
+
+    # Functions and properties we won't support.
+    values = unsupported_property(
+        'values',
+        reason="If you want to collect your data as an NumPy array, use 'to_numpy()' instead.")
+
+    to_pickle = unsupported_function(
+        'to_pickle',
+        reason="Pickled data is not designed for a long term.")
```

Comment:
I'd change this to

"For storage, we encourage you to use Delta or Parquet, instead of Python pickle format."